### PR TITLE
Log cache reference update

### PR DESCRIPTION
**Pull request by rroberts2222-zz**
Update references to archived log-cache to use go-log-cache instead

### CODE REVIEW

File: latency-log-cache/go.mod [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
+	code.cloudfoundry.org/rfc5424 v0.0.0-20180905210152-236a6d29298a // indirect
+	github.com/blang/semver v3.5.1+incompatible // indirect
+	github.com/golang/protobuf v1.5.2 // indirect
+	github.com/grpc-ecosystem/grpc-gateway/v2 v2.6.0 // indirect
+	golang.org/x/net v0.0.0-20210405180319-a5a99cb37ef4 // indirect
+	golang.org/x/sys v0.0.0-20210510120138-977fb7262007 // indirect
+	golang.org/x/text v0.3.5 // indirect
+	google.golang.org/genproto v0.0.0-20210903162649-d08c68adba83 // indirect
+	google.golang.org/grpc v1.40.0 // indirect
+	google.golang.org/protobuf v1.27.1 // indirect
```
Comment (ctlong) [inline latency-log-cache/go.mod:21]: A bit weird to bring in both protobuf packages.. It's indirect though so can't really be helped I guess. Seems like go-loggregator uses github.com/golang/protobuf and go-log-cache uses google.golang.org/protobuf.

Comment (mkocher) [reply latency-log-cache/go.mod:21]: going to ignore this duplication for now since this isn't production code.

Comment (ctlong) [reply latency-log-cache/go.mod:21]: This was a while ago. I've realized since that it's necessary.

**Comment by ctlong** [comment]
@Benjamintf1 are you able to merge this? I don't have write access

**Comment by ctlong** [comment]
🤔 @Benjamintf1 you should have write access according to the [org automation yaml](https://github.com/cloudfoundry/community/blob/1a97f2cec4317fb836440da4ff93ab70cd2cbf5d/org/cloudfoundry.yml#L3493-L3532).

🙏 @ameowlia do you have admin access here? Is there any setting that might prevent approvers with write access from merging PRs?

**Pull request merged**